tests/unit/_internal/bento/test_bento.py

```diff
@@ -81,10 +81,9 @@ def test_bento_info(tmpdir: "Path"):
         assert bentoinfo_b_from_yaml == bentoinfo_b
 
 
-def build_test_bento(model_store: "ModelStore") -> Bento:
+def build_test_bento() -> Bento:
```

Comment:
This `model_store` is actually a fixture under `conftest.py` to create a model_store under pytest tmp test directory